python/ray/data/preprocessors/utils.py

```diff
@@ -7,6 +8,26 @@ def simple_split_tokenizer(value: str) -> List[str]:
     return value.split(" ")
 
 
+def default_tokenization_fn(value: str) -> List[str]:
+    """Split a string into a list of tokens.
+
+    This function converts ``value`` to lowercase, removes punctuation, and then
+    delimits tokens by whitespace.
+
+    Args:
+        value: The string to tokenize.
+
+    Examples:
+        >>> string = "Hello, world!\nReadability counts."
+        >>> default_tokenization_fn(string)
+        ['hello', 'world', 'readability', 'counts']
+    """
+    value = value.lower()
+    for character in string.punctuation:
+        value = value.replace(character, "")
```

Comment:
Should removed punctuation be included in the split?
```suggestion
        value = value.replace(character, " ")
```